### ISSUE TITLE
unit bug forcebalance 1.9.4 + evaluator 0.4.1 + toolkit 0.11.4 in tutorial04 

### ISSUE THREAD

**Issue by wenyan4work** [openforcefield/openff-evaluator#482]
when running forcebalance optimization example in tutorial04, forcebalance raises the following error after the gradients are computed in forcebalance and reported back to forcebalance:

```python
Warning: importing 'simtk.openmm' is deprecated.  Import 'openmm' instead.
Traceback (most recent call last):
  File "/root/mambaforge/envs/fb/bin/ForceBalance.py", line 45, in Run_ForceBalance
    optimizer.Run()
  File "/root/mambaforge/envs/fb/lib/python3.9/site-packages/forcebalance/optimizer.py", line 322, in Run
    xk = self.OptTab[self.jobtype]()
  File "/root/mambaforge/envs/fb/lib/python3.9/site-packages/forcebalance/optimizer.py", line 950, in NewtonRaphson
    return self.MainOptimizer(b_BFGS=0)
  File "/root/mambaforge/envs/fb/lib/python3.9/site-packages/forcebalance/optimizer.py", line 496, in MainOptimizer
    data        = self.Objective.Full(xk,Ord,verbose=True)
  File "/root/mambaforge/envs/fb/lib/python3.9/site-packages/forcebalance/objective.py", line 314, in Full
    Objective = self.Target_Terms(vals, Order, verbose, customdir)
  File "/root/mambaforge/envs/fb/lib/python3.9/site-packages/forcebalance/objective.py", line 255, in Target_Terms
    Ans = Funcs[Order](mvals, customdir=customdir)
  File "/root/mambaforge/envs/fb/lib/python3.9/site-packages/forcebalance/target.py", line 319, in get_H
    Ans = self.meta_get(mvals,1,1,customdir=customdir)
  File "/root/mambaforge/envs/fb/lib/python3.9/site-packages/forcebalance/target.py", line 584, in meta_get
    Answer = self.get(mvals, AGrad, AHess)
  File "/root/mambaforge/envs/fb/lib/python3.9/site-packages/forcebalance/evaluator_io.py", line 739, in get
    estimated_data_set, estimated_gradients = self._extract_property_data(
  File "/root/mambaforge/envs/fb/lib/python3.9/site-packages/forcebalance/evaluator_io.py", line 670, in _extract_property_data
    gradient_value = gradient.value.to(gradient_unit).magnitude
  File "/root/mambaforge/envs/fb/lib/python3.9/site-packages/pint/facets/plain/quantity.py", line 522, in to
    magnitude = self._convert_magnitude_not_inplace(other, *contexts, **ctx_kwargs)
  File "/root/mambaforge/envs/fb/lib/python3.9/site-packages/pint/facets/plain/quantity.py", line 471, in _convert_magnitude_not_inplace
    return self._REGISTRY.convert(self._magnitude, self._units, other)
  File "/root/mambaforge/envs/fb/lib/python3.9/site-packages/pint/facets/plain/registry.py", line 911, in convert
    return self._convert(value, src, dst, inplace)
  File "/root/mambaforge/envs/fb/lib/python3.9/site-packages/pint/facets/context/registry.py", line 393, in _convert
    return super()._convert(value, src, dst, inplace)
  File "/root/mambaforge/envs/fb/lib/python3.9/site-packages/pint/facets/nonmultiplicative/registry.py", line 206, in _convert
    return super()._convert(value, src, dst, inplace)
  File "/root/mambaforge/envs/fb/lib/python3.9/site-packages/pint/facets/plain/registry.py", line 944, in _convert
    raise DimensionalityError(src, dst, src_dim, dst_dim)
pint.errors.DimensionalityError: Cannot convert from 'gram * mole / kilocalorie / milliliter' ([substance] * [time] ** 2 / [length] ** 5) to 'kilogram / meter ** 3' ([mass] / [length] ** 3)
```

**Comment by mattwthompson** [comment]
Right, that version of ForceBalance doesn't work with the most recent version of Evaluator

**Comment by mattwthompson** [comment]
I'm not sure at the moment. I'm pretty sure that changes are only needed there, not here.

**Comment by mattwthompson** [comment]
This might be fixed with 1.9.5, I'm not sure

**Comment by mattwthompson** [comment]
I'm pretty sure it is - please re-open if I'm mistaken
